README.md

```diff
@@ -1 +1,72 @@
 # My-Mobile-Portfolio
+
+This is my Mobile portfolio website where I get to share projects and information about me.
+
+## Built With
+- HTML
+- CSS
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+## Prerequisites
+- Install a modern browser e.g Google Chrome, Safari
+- Have git installed.
+- Have the LTS version of Node.js installed.
+
+## Setup
+- You need to clone the repo using:
+- git clone git@github.com:emarkees/My-Mobile-Portfolio.git
+- Run npm install to download required modules locally
+- In the project directory, you can run:
+
+### npm start
+- Runs the app in the development mode.
+- Open http://localhost:3000 to view it in your browser.
+- The page will reload when changes are made.
+- You may also see any lint errors in the console.
+
+### npm test
+- Launches the test runner in the interactive watch mode.
+- See the section about running tests for more information.
+
+### npm run build
+- Builds the app for production to the build folder.
+- It correctly bundles React in production mode and optimizes the build for the best performance.
+- The build is minified and the filenames include the hashes.
+- Your app is ready to be deployed!
+
+See the section about deployment for more information.
+
+### npm run eject
+Note: this is a one-way operation. Once you eject, you can't go back!
+
+- If you aren't satisfied with the build tool and configuration choices, you can eject at any time. This command will remove the single build dependency from your project.
+
+- Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except eject will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+
+- You don't have to ever use eject. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+
+## Deployment
+Deploy on any web server that can host a Node.js runtime environment.
+
+## Author
+👤Adeyemi OLusola
+
+- GitHub: [@emarkees](https://github.com/emarkees)
```

Comment:
- [ ] Please make sure to use the new Microverse readme template that you can find [here](https://github.com/microverseinc/readme-template).